Directory.Build.props

```diff
@@ -160,6 +162,7 @@
     <_packageRID Condition="$(TargetGroup.EndsWith('aot'))">win10-$(ArchGroup)-aot</_packageRID>
     <PackageRID Condition="'$(PackageRID)' == ''">$(_packageRID)</PackageRID>
     <PackageRID Condition="'$(PackageRID)' == ''">$(RuntimeOS)-$(ArchGroup)</PackageRID>
+    <MicrosoftNetCoreIlasmPackageRuntimeId Condition="'$(MicrosoftNetCoreIlasmPackageRuntimeId)' == ''">$(PackageRID)</MicrosoftNetCoreIlasmPackageRuntimeId>
```

Comment:
Added this, so that the `ilasm` package we copy from matches what the custom restore targets will use.